database/storage.go

```diff
@@ -137,9 +126,54 @@ func (s *Storage) rows(table string) (*sql.Rows, error) {
 }
 
 func (s *Storage) nRows(table string, n string) (*sql.Rows, error) {
-	nRows, err := s.conn.Query(fmt.Sprintf("SELECT * FROM `%s` LIMIT %s", table, n))
+	column, err := s.primaryColumn(table)
+	if err != nil {
+		return nil, err
+	}
+	nRows, err := s.conn.Query(fmt.Sprintf("SELECT * FROM `%s` ORDER BY %s DESC LIMIT %s", table, column, n))
```

Comment:
In this case I would extract a string var
```go
query := fmt.Sprintf("SELECT * FROM `%s` ORDER BY %s DESC LIMIT %s", table, column, n)
nRows, err := s.conn.Query(query)
```

Reply:
@rafaeljesus @mereba sprintf is vulnerable to SQL Injection, please use prepared statements / parametrized queries.
seems like `Query()` accepts parameters and will do the "concatenation" for you, https://golang.org/pkg/database/sql/#DB.Query